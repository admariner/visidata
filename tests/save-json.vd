sheet	col	row	longname	input	keystrokes	comment
	override	overwrite	set-option	always		^S errors in batchmode, if trying to save over an existing file
			open-file	sample_data/test.jsonl	o	
test			columns-sheet		C	
test_columns	name		sort-asc		[	
test_columns			quit-sheet		q	
test	amt		key-col		!	
test	key1		key-col		!	
test			sort-keys-asc		g[	
test	amt		key-col		!	
test	key1		key-col		!	
