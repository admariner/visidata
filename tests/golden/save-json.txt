	amt	key1	key2	qty
#ERR	#ERR	#ERR	#ERR	#ERR
		2016-01-01 11:00:00	foo	1
		2018-07-27 16:44	bar	
	.01	2018-07-27 18:44	baz	256
	.01	2018-10-20 18:44	foo	30.0
	.3	2017-12-25 8:44	bar	16
	3	2016-01-01 1:00		2
	3.3		baz	32
	43.2		baz	4
	9.1	2018-07-27 4:44		64
